### PR TITLE
add Speech API auto punctuation sample

### PR DESCRIPTION
**Pull request by dizcology**

### CODE REVIEW

**Comment by telpirion** [comment]
The auto punctuation sample helps greatly. Thank you!

File: speech/cloud-client/beta_snippets.py [hunk 1]
```diff
@@ -99,6 +100,32 @@ def transcribe_file_with_metadata(path):
 # [END speech_transcribe_file_with_metadata]
 
 
+# [START speech_transcribe_file_with_auto_punctuation]
+def transcribe_file_with_auto_punctuation(path):
+    """Transcribe the given audio file using an enhanced model."""
```
Comment (nnegrey) [inline speech/cloud-client/beta_snippets.py]: the "enhanced model" bit, this is where I'm confused between the one type of enhanced models and the model selected enhanced models. (Not sure if this point really matters though)

Comment (dizcology) [reply speech/cloud-client/beta_snippets.py]: I failed to update the docstring - fixing it now.

**Comment by jgeorge-burwood** [comment]
Is this functionality available in python client library 0.33.0?  Trying to test  but its coming back with:  Protocol message RecognitionConfig has no "enable_automatic_punctuation" field.

**Comment by jgeorge-burwood** [comment]
solved -- I needed the types and enums from v1p1beta1 as well.  doh!

**Comment by dalmam11** [comment]
jgeorge-burwood I have the same issue where did you found v1p1beta1?

**Comment by dalmam11** [comment]
cannot import name 'speech_v1p1beta1'

**Comment by jgeorge-burwood** [comment]
dalmam11 make sure you're using at least 0.33.0 of the client library.   
pip install --upgrade google-cloud-speech

Then just add the imports, you'll probably want something like:
...
from google.cloud import speech_v1p1beta1 as speech
from google.cloud.speech_v1p1beta1 import enums
...

**Pull request merged**